File: v2/lua/audio/main.lua
local json = require('cjson')
local mime = require("mime")
local hc = require('lib.httpclient').new()

local url = 'http://api-audio-bj.fengkongcloud.com/v2/saas/anti_fraud/audio'
local accessKey = '{ACCESS_KEY}'

local inp = assert(io.open('../files/demo.pcm', 'rb'))
local content = mime.b64(inp:read('*all'))

local payload = json.encode({
  accessKey = accessKey,
  type = 'DEFAULT',
  appId = 'default',
  btId = '{BT_ID}',
  callback = 'https://jsonplaceholder.typicode.com/posts/',
  data = {
    -- url 和 content 至少提供一个
    -- url = 'https://jsonplaceholder.typicode.com/audio',
    content = content,
    formatInfo = {
      format = 'pcm',
      rate = 8000,
      track = 1,
    }
  }
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])

File: v2/python/audio/main.py
import base64

import requests

url = 'http://api-audio-bj.fengkongcloud.com/v2/saas/anti_fraud/audio'
access_key = '{ACCESS_KEY}'
bt_id = '{BT_ID}'
filename = '../files/demo.pcm'

with open(filename, 'rb') as f:
    content = base64.b64encode(f.read())

payload = {
    'accessKey': access_key,
    'type': 'DEFAULT',
    'appId': 'default',
    'btId': bt_id,
    'callback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        # url 和 content 至少提供一个
        # 'url': 'https://jsonplaceholder.typicode.com/audio',
        'content': content.decode('utf-8'),
        'formatInfo': {
            'format': 'pcm',
            'rate': 8000,
            'track': 1,
        },
    },
}

res = requests.post(url, json=payload)
# 通过 code 和 riskLevel 判断返回，具体请参考接口文档
if res:
    print(res.json())
